test/task/mock/MockTarget.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.15;
+
+import {Test} from "forge-std/Test.sol";
+
+contract MockTarget is Test {
+    address public task;
+
+    function setTask(address _task) public {
+        task = _task;
+    }
+
+    function setSnapshotIdTask(uint256 id) public {
+        vm.store(task, bytes32(uint256(48)), bytes32(id));
```

Comment:
Like the other slots, can we make this into a variable with a name. 

I'm getting:
```
Name                       | Type                                                   | Slot | Offset | Bytes | Contract                                             
_startSnapshot             | uint256                                                | 48   | 0      | 32    | test/task/mock/MockMultisigTask.sol:MockMultisigTask
```